examples/chip-tool/commands/common/CommandInvoker.h

```diff
@@ -107,7 +107,7 @@ class CommandInvoker final : public ResponseReceiver<typename RequestType::Respo
     }
 
     CHIP_ERROR InvokeGroupCommand(Messaging::ExchangeManager * exchangeManager, FabricIndex fabric, GroupId groupId,
-                                  const RequestType & aRequestData)
+                                  NodeId sourceNodeId, const RequestType & aRequestData)
```

Comment:
What do you think of changing `SessionManager::Init` to take a pointer to the fabric table and store it?  Then whoever sets it up (Server and ControllerFactory) can provide that pointer and it can use the fabric table sanely.

Reply:
Discussed of this with @tcarmelveilleux this morning, and problem is that in some case you will need a SessionManager without any fabric table. Adding a fabric table pointer was one of the option I considered but given the requirement that it would imply, this solution (passing nodeId) seemed the safest dependency wise.

Reply:
OK.  Let's do this for now, but work on fixing group sessions to not be attached to session manager at all, and then we can fix these things to be saner.